language/e2e-tests/src/tests/data_store.rs

```diff
@@ -0,0 +1,415 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
all those tests fail in master without the change showing the problem described in the summary.
Essentially the blocks with multiple transactions all succeed!!